packages/front-end/components/Layout/Layout.tsx

```diff
@@ -122,6 +122,22 @@ const navlinks: SidebarLinkProps[] = [
       },
     ],
   },
+  {
+    name: "Integrations",
+    href: "/integrations",
+    Icon: BsPlug,
+    path: /^(integrations)/,
+    autoClose: true,
+    feature: "slack-integration",
```

Comment:
Only on for those where `slack-integration` is true. Right now this is configured off in production, on in dev, with a 50% rollout.